content/en/docs/languages/python/propagation.md

```diff
@@ -9,12 +9,29 @@ Although not limited to tracing, it is what allows traces to build causal
 information about a system across services that are arbitrarily distributed
 across process and network boundaries.
 
-## Manual W3C Trace Context Propagation
+OpenTelemetry provides a text-based approach to propagate context to remote
+services using the [W3C Trace Context](https://www.w3.org/TR/trace-context/)
+HTTP headers.
 
-The following generic example demonstrates how you can propagate trace context
+## Automatic context propagation
+
+Auto-instrumentation exists for some popular Python frameworks.
```

Comment:
It would be nice to validate this and add examples. I couldn't find any mention of this except for Celery...

Reply:
sure, will take a look

Reply:
check done, this is true. From the statistic result of March, based on download count, the popular instrumentation are 

project | downloads
-- | --
urllib3 | 471792853
requests | 392426582
jinja2 | 151665370
sqlalchemy | 104640315
aiohttp | 102008929
flask | 97099707
httpx | 46978770
fastapi | 45061219
redis | 39668426
starlette | 29139254
pymongo | 24734312
psycopg2 | 15699913
django | 14132946
celery | 9053682
psycopg | 5138183
asyncpg | 3879504
pymemcache | 1314158
falcon | 1103115